sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -514,6 +578,116 @@ static void _netif_list(kernel_pid_t iface)
     puts("");
 }
 
+static int _netif_set_u32(kernel_pid_t iface, netopt_t opt, uint32_t context,
+                          char *u32_str)
+{
+    unsigned long int res;
+    bool hex = false;
+
+    if (_is_number(u32_str)) {
+        if ((res = strtoul(u32_str, NULL, 10)) == ULONG_MAX) {
+            puts("error: unable to parse value.\n"
+                 "Must be a 32-bit unsigned integer (dec or hex)\n");
+            return 1;
+        }
+    }
+    else {
+        if ((res = strtoul(u32_str, NULL, 32)) == ULONG_MAX) {
+            puts("error: unable to parse value.\n"
+                 "Must be a 32-bit unsigned integer (dec or hex)\n");
+            return 1;
+        }
+
+        hex = true;
+    }
+
+    if (res > 0xffffffff) {
```

Comment:
`UINT32_MAX`? Also, I think it is safe to make this an assert for now. I don't know of any platform in our scope where `sizeof(unsigned long) > sizeof(uint32_t)`.

Reply:
Done